bindings/cpp/session.cpp

```diff
@@ -2233,12 +2248,12 @@ error_info session::mix_states(const key &id, std::vector<int> &groups)
 }
 
 async_iterator_result session::start_iterator(const key &id, const std::vector<dnet_iterator_range>& ranges,
-								uint32_t type, uint64_t flags,
-								const dnet_time& time_begin, const dnet_time& time_end)
+		uint32_t type, uint64_t flags, const dnet_time& time_begin, const dnet_time& time_end)
```

Comment:
It will return an error if there is server-send iterator flag and there are no groups.
New flags are safe for old iterators.

We could clear those flags, but what's the purpose? Blind clearing is wrong - if client has an error and sets up server-send iterator via this call, clearing flags will result in starting usual data iterator, client will think that his data has been copied. I would rather return error for this case. But using other flags is safe.